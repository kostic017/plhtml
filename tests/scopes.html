<!doctype html>
<html lang="en">
    <head>
        <title>`Scopes Test`</title>
    </head>
    <body>

        <main>

            <var class="integer">a</var>
            <data value="1">a</data>
            <output>a + `\n`</output>

            <div data-if="true">

                <div data-if="true">

                    <var class="integer">a</var>
                    <data value="3">a</data>
                    <output>a + `\n`</output>

                </div>

                <output>a + `\n`</output>

                <var class="integer">a</var>
                <data value="2">a</data>
                <output>a + `\n`</output>

            </div>

        </main>

    </body>
</html>
