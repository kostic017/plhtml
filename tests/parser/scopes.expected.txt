"Scopes Test"
Main:
    integer a
    a = 1
    Write: a + "\n"
    If true
        If true
            integer a
            a = 3
            Write: a + "\n"
        Write: a + "\n"
        integer a
        a = 2
        Write: a + "\n"
